Fix error message on activation

sinukaarel opened the pull request:
Fixes #14 

No more error messages when activating plugin.
`show tables like` query returned error and with that a `null` value. That caused creating tables even though tables were already created.

Ideal solution would be that if we switched to `dbDelta()` functionality but that means we no loger can use `key` as a column name. This workload is much bigger. This was the simplest solution I found.

kaittodesk commented:
Wouldn't it be better if backticks and quotes would be removed from the `SHOW TABLES` query entirely? It should be safe as `esc_sql` is already used to sanitize table name.

sinukaarel commented:
In the last commit 94c1f4f212e6489d69f09f02589183f0a3076862 I have changed quite a bit and looked more into table creation and wordpress `prepare()` functionality.

Things I found out are that :
- You can't use` prepare()` functionality in creating a table. It is meant to escape string values and those are automatically quoted. So in the case of `$table_name` and `$charset_collate` it brakes sql statement.
```
CREATE TABLE 'wp_sendsmaily_config' (
```
- I have removed the whole `preq_replace(`) thing and concatenate sql statement so no extra spaces will be added
- I only keep backticks around `key` column name value so that it won'y be considered as sql keyword. Other column names should be okay and don't need backticks
https://github.com/sendsmaily/sendsmaily-wordpress-plugin/blob/94c1f4f212e6489d69f09f02589183f0a3076862/includes/activator.php#L28-L35
- `dbDelta()` functionality can be used as Wordpress requirements state on`sendsmaily_autoresp` table creation and manipulation. It clears up that part quite a lot.

This clears up activation error message #14 . 
I think switching to `dbDelta()` functionality on first table should be considered in next major update as it might not be backwards compatible due to table structural change. ( Changing `key` column name is required)
